chromadb/db/duckdb.py

```diff
@@ -82,6 +82,10 @@ def create_collection(
         dupe_check = self.get_collection(name)
         if len(dupe_check) > 0:
             if get_or_create is True:
+                if dupe_check[0][2] != metadata:
+                    self.update_collection(name, new_name=name, new_metadata=metadata)
```

Comment:
Additionally - will this be the same behavior as upsert with embedding metadata?

Reply:
I hope so. That won't get merged until after this, so I think we should argue that "replace" makes more sense than "merge" across the board.

Reply:
Ok great. Let's make sure that what we do.